Collector/ProfilePlugin.php

```diff
@@ -58,7 +58,9 @@ public function handleRequest(RequestInterface $request, callable $next, callabl
     {
         $profile = new Profile($this->pluginName, $this->formatter->formatRequest($request));
 
-        $this->collector->getCurrentStack()->addProfile($profile);
+        if (false !== $stack = $this->collector->getCurrentStack()) {
```

Comment:
Thank you for the reivew. You are correct. I should update. 

We are trying out https://app.assignees.io/. A service that automatically finds the best reviewer. Somehow it selected you.